SINGLE_CONNECTION.md

```diff
@@ -0,0 +1,195 @@
+isolation=level | The default transaction isolation level for the connection to use. Isolation levels supported are consistent_read and read_committed. Note that these values are case sensitive. For more information, see CONSISTENT READ and READ COMMITTED.
+keepAlive=[ true \| false ] | Allows enabling of TCP KeepAlive on the connection from client to Transaction Engine. If not specified, keepAlive defaults to false. When keepAlive=true, the driver enables TCP KeepAlive on the socket that it opens to the Transaction Engine. 
+LBPolicy=valueA | From the client application, this property allows selective load balancing across Transaction Engines. For information on using LBPolicy, see Load Balancer Policies. 
+LBQuery=\<selector>(\<filter>) | From the client application, this property allows selective load balancing across Transaction Engines. For information on using LBQuery, see Load Balancer Policies.
+lock-wait-timeout=sec | Configures lock wait time (in seconds) when establishing a new connection. For more information, see About the NuoDB SQL Configuration File. The default is equal to the system property DEFAULT_LOCK_WAIT_TIMEOUT. For more information, see SQL System Properties. 
+memory-limit-bytes=size | The memory limit (per connection) in bytes for all blocking SQL engine operations: Hash-grouping, sorting, sorting via priority queue for limit, distinct (via hash-grouping), union, listagg, table functions, and stored procedures returning result set(s) that accumulate data - in main memory. To run statements with a memory limit different from the system-wide limit, start a new connection with this connection property set to the per-connection limit. If you do not define a value for memory-limit-bytes, the value set for the DEFAULT_CONNECTION_MEMORY_LIMIT system property is the default setting used.
+password=pwd | The password for the user account used to connect to the database. 
+PreferInternalAddress=[ true \| false ] | Use internal address and port for TE, as specified by the external-address and external-port process labels. Default is false (disabled). For a detailed explanation, see Controlling Connectivity, a special case of using Load Balancer Policies with labels. 
+rollbackMode=[ procedure \| transaction \| off ] | Controls the behavior of transactions executed inside a stored procedure. The default is off. See About Implicit and Explicit Transactions.
+schema=name | The default schema that the connection should use when objects (such as tables and views) are not fully qualified.
+TimeZone=timezone | The default time zone for the connection to use (this is also known as the session time zone). If not specified, the application’s default time zone is used.
+user=name | The user name for connecting to the database.
+verifyHostname=[ true \| false ] | Verifies the DN name of the SSL server against the name presented on the certificate. The default is true.
+
```

Comment:
A lot of these property explanations end with "For more information see <XYZ>"
I think these should be removed, or a link to the proper information be added.